### PR TITLE
update config and version for v0.4

### PR DESCRIPTION
**Pull request by bcumming**

### CODE REVIEW

**Comment by bcumming** [comment]
bors try.

File: setup.py [hunk 1]
```diff
@@ -154,14 +154,15 @@ def run(self):
...
+    author='The Arbor dev team.',
     url='https://github.com/arbor-sim/arbor',
     description='High performance simulation of networks of multicompartment neurons.',
     long_description=long_description,
     long_description_content_type='text/markdown',
     classifiers=[
-        'Development Status :: 4 - Beta', # Upgrade to "5 - Production/Stable" on release of v0.4
+        'Development Status :: 5 - Production/Stable',
         'Intended Audience :: Science/Research',
+        "License :: OSI Approved :: BSD License",
```
Comment (thorstenhater) [inline setup.py:165]: Stylistic choice: The rest of items uses ', this one ".

File: setup.py [hunk 2]
```diff
@@ -154,14 +154,15 @@ def run(self):
...
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
```
Comment (thorstenhater) [inline setup.py:168]: I do not know whehter we should add C++/CUDA here, eg numpy adds C to the list.

**Review by thorstenhater — COMMENTED** [review]
Two small comments, rest looks good.

File: setup.py (hunk 2 above)
Comment (bcumming) [reply setup.py:168]: We could. I find these things really fiddly to get working properly.

File: setup.py (hunk 1 above)
Comment (bcumming) [reply setup.py]: Fixed.

File: setup.py (hunk 2 above)
Comment (bcumming) [reply setup.py:168]: I added C++ and CUDA 10 & 11

Comment (thorstenhater) [reply setup.py:168]: 👍

**Review by thorstenhater — APPROVED** [review]
Looks good.

**Pull request merged**